src/main/java/seedu/taassist/logic/commands/ClearCommand.java

```diff
@@ -18,6 +19,10 @@ public CommandResult execute(Model model) {
         requireNonNull(model);
         boolean isInFocusMode = model.isInFocusMode();
         model.setTaAssist(new TaAssist());
-        return new CommandResult(MESSAGE_SUCCESS, false, false, false, isInFocusMode);
+        if (isInFocusMode) {
+            return new CommandResult(MESSAGE_SUCCESS, UI_UNFOCUS);
```

Comment:
When using constants from other class, it is convention (and more clear) to qualify using the class name (though it is not properly enforced throughout the codebase, there are stray constant imports too). I think at least for Enum we would do this, if not for other constants.
```suggestion
            return new CommandResult(MESSAGE_SUCCESS, UiAction.UI_UNFOCUS);
```

Reply:
Sure!